apps/portlet-documents/src/main/webapp/vue-app/attachment/components/snackbar/AttachmentsNotificationAlerts.vue

```diff
@@ -45,7 +45,9 @@ export default {
   },
   methods: {
     addAlert(alert) {
-      if (alert) {
+      //check if the alert is already displayed
+      const alertExists = this.alerts.some((existingAlert) => existingAlert.message === alert.message && existingAlert.type === alert.type);
```

Comment:
Will this disable also alerts for successful file uploads too for example ?

Reply:
yes this check to no display any attachment alert already displayed in this moment !

Reply:
could you check if this is expected already ? I believe we need to have as many alerts as needed when uploading files .